src/elements/mixin/spintransport.H

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Chad Mitchell
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_ELEMENTS_MIXIN_SPIN_TRANSPORT_H
+#define IMPACTX_ELEMENTS_MIXIN_SPIN_TRANSPORT_H
+
+#include "particles/ImpactXParticleContainer.H"
+
+#include <ablastr/constant.H>
+
+#include <AMReX_Math.H>
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
```

Comment:
for `std::sqrt` let us include:
```suggestion
#include <AMReX_REAL.H>

#include <cmath>

```